client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -219,14 +218,25 @@ const PatternAssembler = ( props: StepProps & NoticesProps ) => {
 			pattern_categories: categories.join( ',' ),
 			category_count: categories.length,
 			pattern_count: patterns.length,
+			page_slugs: ( pageSlugs || [] ).join( ',' ),
 		} );
+
 		patterns.forEach( ( { ID, name, category } ) => {
 			recordTracksEvent( PATTERN_ASSEMBLER_EVENTS.PATTERN_FINAL_SELECT, {
 				pattern_id: ID,
 				pattern_name: name,
 				pattern_category: category?.name,
 			} );
 		} );
+
+		pages.forEach( ( { ID, name, categories = {} } ) => {
+			const category_slug = Object.keys( categories )[ 0 ];
+			recordTracksEvent( PATTERN_ASSEMBLER_EVENTS.PAGE_FINAL_SELECT, {
+				page_id: ID,
```

Comment:
Ultimately, pages are still patterns, what about keeping the "shape" of these params the same as pattern selection? (`pattern_id`, `pattern_name`, `pattern_category`)

`page_id` sounds misleading to me as it's not the ID of a WordPress page...

Reply:
Makes sense to me 🙂
![Screenshot 2023-12-18 at 12 00 32 PM](https://github.com/Automattic/wp-calypso/assets/797888/d4ccaa78-7fb0-4c60-ab37-c9cf4c280718)